capstone-backend/src/main/java/com/google/sps/data/BookList.java

```diff
@@ -17,77 +18,85 @@
  */
 public final class BookList {
```

Comment:
If you've pulled BaseEntity into this PR, could be a good time to have BookList extend BaseEntity, and remove the use of UUID to set the id.